UOP1_Project/Assets/Scripts/CameraManager.cs

```diff
@@ -9,6 +9,15 @@ public class CameraManager : MonoBehaviour
 	public InputReader inputReader;
     public CinemachineFreeLook freeLookVCam;
 
+ 	[Tooltip("General multiplier for camera sensitivity/speed")]
+	public float cameraSensitivity = 7.0f;
+
+ 	private void Awake()
```

Comment:
I put this in `Awake` instead of `Update` because

1. I figured this shouldn't be changing.
2. I figured it'd save us from doing one more thing each frame.

Reply:
It's fine to have it here indeed, here's a reference to the Docs about Awake: https://docs.unity3d.com/ScriptReference/MonoBehaviour.Awake.html

Specifically the sentence: "Use Awake to initialize variables or states before the application starts." Applies directly here :)